src/index.tsx

```diff
@@ -3,12 +3,38 @@ import "./assets/js/uswds-init.min.js";
 import "./assets/js/uswds.min.js";
 import "./assets/theme/styles.scss";
 
+import { ActionsMenu } from "./components/Header/ActionsMenu";
 import { Button } from "./components/Button/Button";
 import { Card } from "./components/Card/Card";
+import { Checkbox } from "./components/Checkbox/Checkbox";
+import { Dropdown } from "./components/Dropdown/Dropdown";
+import { Footer } from "./components/Footer/Footer";
+import { Header } from "./components/Header/Header";
 import { Icon } from "./components/Icon/Icon";
-import { Logo } from "components/Logo/Logo";
-import { TextArea } from "components/TextArea/TextArea";
-import { TextInput } from "components/TextInput/TextInput";
```

Comment:
The absolute imports in this file break the components when using the NPM package. Added a leading `./` where needed.